src/Symfony/Component/Translation/Dumper/JsonFileDumper.php

```diff
@@ -23,9 +23,23 @@ class JsonFileDumper extends FileDumper
     /**
      * {@inheritdoc}
      */
+    public function formatCatalogue(MessageCatalogue $messages, $domain, array $options = array())
```

Comment:
@aitboudad yes, you are right, change it to protected
@ogizanagi yep, change it to JSON_HEX_QUOT, it's from 5.3.